### ISSUE TITLE
Creating the Dashboard and other pages

### ISSUE THREAD

**Issue by SajjaPremsai** [LakshmiSowmya04/GradLink#47]
## Creating pages

Hi this is Prem i have checked the project and runs on my local. There are few pages need to develop if no one is doing iam interested developing those pages.

**Comment by HarzhMehta** [comment]
Id like to work on the same

**Comment by HarzhMehta** [comment]
I have started working on login and signup pages.... you can start dashboard maybe ? Can you join the discord grp

**Comment by SajjaPremsai** [comment]
> I have started working on login and signup pages.... you can start dashboard maybe ? Can you join the discord grp

Yeah okay i can start the dashboard. Yeah i have joined the discord grp

**Comment by HarzhMehta** [comment]
👍
